pkg/diagnostics/component_monitoring.go

```diff
@@ -31,6 +31,13 @@ const (
 	CryptoOp                 = "crypto_op"
 )
 
+type ConversationType string
+
+const (
+	NonStreamingConversation ConversationType = "non_streaming"
+	StreamingConversation    ConversationType = "streaming"
```

Comment:
Please use `unary` or some other verbiage to describe this mode, rather than the negation of the other mode.